src/packages/@ncigdc/modern_components/AffectedCasesTable/AffectedCasesTable.js

```diff
@@ -363,9 +363,9 @@ const Component = compose(
                     age_at_diagnosis: ageDisplay(diagnosis.age_at_diagnosis),
                     tumor_stage: diagnosis.tumor_stage,
                     days_to_last_follow_up: (diagnosis.days_to_last_follow_up ||
-                      0)
+                      '--')
```

Comment:
`days_to_last_follow_up:0` is in the response so yea I think 0 is a real data point but `--` should be used is `days_to_last_follow_up` is `null`